03/app.js

```diff
@@ -1,2 +1,10 @@
 const n = 24;
-const oddNumbers = [];
\ No newline at end of file
+const oddNumbers = [];
+
+for (i=0; i<=n; i++) {
+    if (i%2 === 1) {
+        oddNumbers.push(i);
+    }
+}
```

Comment:
Spróbuj teraz rozwiązać to zadanie przy pomocy `.fitler()` w taki sposób, że wygeneruj najpierw tablicę ze wszystkimi liczbami, a potem wygeneruj kolejną tylko zwierającą liczby nieparzyste.